<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changepage</class>
 <widget class="QWidget" name="changepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>49</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(125, 116, 255);</string>
   </property>
   <property name="text">
    <string>  名 字</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(131, 234, 255);</string>
   </property>
   <property name="text">
    <string>  时 间 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(221, 255, 110);</string>
   </property>
   <property name="text">
    <string>  类 型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 153, 185);</string>
   </property>
   <property name="text">
    <string>  金 额</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="change_name_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: rgb(125, 116, 255);
   border-radius: 15px; /* 边框圆角 */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="change_time_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color: rgb(131, 234, 255);
   border-radius: 15px; /* 边框圆角 */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="change_kind_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: rgb(221, 255, 110);
   border-radius: 15px; /* 边框圆角 */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="change_money_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color: rgb(255, 153, 185);
   border-radius: 15px; /* 边框圆角 */
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="make_sure">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
